src/com/mellanox/cicd/Matrix.groovy

```diff
@@ -427,13 +428,15 @@ def buildDocker(image, config) {
                 config.logger.info("Forcing building file per user request: ${filename} ... ")
                 need_build++
             }
-            if (config.get("cFiles").contains(filename)) {
+            config.logger.debug("Dockerfile name: '${filename}'")
```

Comment:
IMO not needed extra quotes in ``` 'str'```, it is bad for copy&paste

Reply:
Sure, I've just tried to detect extra spaces. In fact it's not the case. I'll remove single quotes.